Tests/VisualRecognitionV3Tests/VisualRecognitionTests.swift

```diff
@@ -1130,4 +1236,253 @@ class VisualRecognitionTests: XCTestCase {
+//                    if image.imageFile == imageFile {
+//                        expectation.fulfill()
+//                        return
+//                    }
+//                }
+//                XCTFail("Image was not successfully added to the collection.")
+//        }
+//        waitForExpectations()
+//    }
+    
+    /** List images in a collection. */
+    func testCreateDeleteCollection() {
+        let description = "Create a collection."
+        let expectation = self.expectation(description: description)
+        
+        let name = "swift-sdk-unit-test-2"
+        var collectionID: String?
+        
+        visualRecognition.createCollection(collectionName: name, failure: failWithError) { collection in
+            XCTAssertEqual(collection.name, name)
+            collectionID = collection.collectionID
+            expectation.fulfill()
+        }
+        waitForExpectations()
+        
+        let description2 = "Delete collection."
+        let expectation2 = self.expectation(description: description2)
+        
+        guard let collection = collectionID else {
+            return
+        }
+        visualRecognition.deleteCollection(collectionID: collection) {
+            expectation2.fulfill()
+        }
+        waitForExpectations()
+    }
+    
+    /** List images in the test collection. */
+    func testListImagesInCollection() {
+        let description = "List images in collection."
+        let expectation = self.expectation(description: description)
+        let failure = { (error: Error) in XCTFail("Could not list images in collection.") }
+        let imageFile = "sign.jpg"
+        
+        visualRecognition.getImagesInCollection(collectionID: collectionID!, failure: failure) { images in
+            XCTAssertEqual(2, images.count)
+            for image in images {
+                if image.imageFile == imageFile {
+                    expectation.fulfill()
+                    return
+                }
+            }
+            XCTFail("Could not list images in collection.")
+        }
+        waitForExpectations()
+    }
+
+    /** Find similar images using the default classifier and all default parameters. */
+    func testSimilarImages() {
+        let description = "Find images similar to an uploaded image using the default classifier."
+        let expectation = self.expectation(description: description)
+        
+        let imageFile = "obama.jpg"
+        
+        visualRecognition.findSimilarImages(
+            withinCollection: collectionID!,
+            imageFile: obama,
+            failure: failWithError) { similarImages in
+                XCTAssertEqual(imageFile, similarImages.imageFile)
+                XCTAssertNotEqual(0, similarImages.similarImages.count)
+                XCTAssertEqual(1, similarImages.imagesProcessed)
+                for image in similarImages.similarImages {
+                    XCTAssertNotEqual(0, image.score)
```

Comment:
Could this value ever be 0? Perhaps we can check and make sure that the image.score isn't `nil` rather than checking if it's a specific value.